tensorflow/core/ops/array_ops.cc

```diff
@@ -665,7 +665,9 @@ y: a tensor of the same shape and type as x but filled with zeros.
 REGISTER_OP("OnesLike")
     .Input("x: T")
     .Output("y: T")
-    .Attr("T: {float, double, int32, int64, complex64, complex128}")
+    .Attr(
```

Comment:
Is there a unit test that a ones_like(boolean tensor) actually runs?

Reply:
The unit test for bool (and uint16) was not there so I updated the unit tests (included in `tensorflow/python/kernel_tests/constant_op_test.py`) and it should be covered now.